src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
ok, i added the missing other references to npsp labels.  not sure why thought I wasn't using lblStreet, which i see still used.  I didn't encounter any markup to $Label.c.Street.